src/doc/rustdoc/src/linking-to-items-by-name.md

```diff
@@ -76,19 +96,66 @@ struct Foo {}
 fn Foo() {}
 ```
 
+The following prefixes can be used:
```

Comment:
I'd collapse this into a single sentence. Most people don't need to care about this, and rustdoc helps with diagnostics when they do.